contrib/cirrus/logformatter

```diff
@@ -440,7 +440,7 @@ END_HTML
         }
     }
 
-    # If Cirrus magic envariables are available, write a link to results.
+    # If Cirrus magic envvariables are available, write a link to results.
```

Comment:
"envariable" has been standard UNIXspeak for decades. I've never seen nor heard of it with two 'v's; the usages I find on DDG all show it in mixed caps: 'envVariable'. This is not the case here. I request that you keep this and all other instances ('envars', 'stdenv') as they are please.

Reply:
Really?
* https://www.google.com/search?q=%22envvariables%22 has About 15,400 results (0.37 seconds) - Did you mean: "env variables"
* https://www.google.com/search?q=%22envariables%22 has About 3,070 results (0.36 seconds)  - Did you mean: "env variables"

I'm not a newbie. I'm happy to drop anything in order to get whatever people will accept, but I don't believe that your assertion about this word is accurate.

Reply:
Yes, really. Did you look at the results of your searches? In the first few pages of hits for the two-vee form, all (except for a few illiterati, and some MS-DOS) are EnvVariable or envVariable -- mixed case. 

I stand by my assertion. In nearly forty years of UNIX programming, I have never seen the two-vee form used in actual code. That is not per se an arbiter of truth -- I could be wrong, and/or the world may have changed -- but just going by the two links you cite, the one-vee sources are more UNIX-literate (apache, MHonArc, stackexchange) than the all-lower-case two-vee instances ("codegrepper", macports.org).